rpcs3/Emu/Cell/SPUThread.h

```diff
@@ -258,7 +258,7 @@ struct spu_channel_4_t
 
 		sync_var.atomic_op([&out_result, &out_value, &out_count, last_value](sync_var_t& data)
 		{
-			if ((out_result = data.count != 0))
+			if (out_result = (data.count != 0))
```

Comment:
I added additional braces around assignment to silence the gcc warning. No really need to change anything here.